sal_interfaces/HVAC/HVAC_Telemetry.xml

```diff
@@ -1,256 +1,1010 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
-<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
-<SALTelemetry>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
     <Subsystem>HVAC</Subsystem>
-    <Version>3.8</Version>
-    <Author></Author>
-    <EFDB_Topic>HVAC_lowerAHUStatus</EFDB_Topic>
+    <EFDB_Topic>HVAC_topic_(mqtt)_(todas_variables_)</EFDB_Topic>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_lsst_fwdsl_piso01_fwdsl_tcc-p1-sala-maquinas</EFDB_Topic>
+    <item>
+      <EFDB_Name>lsst_fwdsl_piso01_fwdsl_vin_01_fwdsl_estado_funcionamiento</EFDB_Name>
+      <Description>lsst_fwdsl_piso01_fwdsl_vin_01_fwdsl_estado_funcionamiento</Description>
+      <IDL_Type/>
```

Comment:
IDL_Type must be specified. This is the data type that DDS will use to transmit the data, so it is fundamental. (Perhaps DDS or the SAL IDL generator has a default, but if so, I have no idea what it might be).

I hope MQTT makes the information directly available (e.g. via some kind of introspection). But if not, then you can work it out based on the types of the fields in the MQTT messages. The problem with doing that is if MQTT reports the fields as standard Python data types (as opposed, say, to ctypes or numpy types) then that loses information: you can't tell the difference between different types of int, nor the difference between float and double.

The available XML data types are described here:
https://confluence.lsstcorp.org/display/LTS/SALPY+Interfaces
note that many of them map to the same data types in IDL, so there are fewer actual choices than it may look like at first.

Reply:
I made sure that (for now) all IDL_Types are set to int, until I receive the proper type info from DatControl.